sube/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(async_closure)]
```

Comment:
Needed? we should move away form nightly only features, we can actually start moving to use stable rust.